src/main/resources/db/changesets/20200102_loadtables.yaml

```diff
@@ -0,0 +1,69 @@
+databaseChangeLog:
+  - changeSet:
+      id: load_tables
+      author: dd
+      changes:
+        - createTable:
+            tableName: load
+            remarks: |
+              One row per unique load tag. Used to lock the load; only one thread operating on a load tag at a time.
+            columns:
+              - column:
+                  name: id
+                  type: ${uuid_type}
+                  defaultValueComputed: ${uuid_function}
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: load_tag
+                  type: text
+                  constraints:
+                    unique: true
+                    nullable: false
+              - column:
+                  name: locked
+                  type: boolean
+                  constraints:
+                    nullable: false
+              - column:
+                  name: locking_flight_id
+                  type: text
+
+        - createTable:
+            tableName: load_file
```

Comment:
Any concerns with the size this table might grow to?

Reply:
Yes. In our discussion we were thinking we would put a limit on the number of files in a given load. The maximum size of the table would be `max-concurrent-loads * max-files-in-a-load`.

Reply:
I wrote a section in the design doc about performance and scaling. Here: https://docs.google.com/document/d/1RPYuQ8gp8ipO1Vy4RfX84vbdyg_wIFO6syckuurTZbU/edit?folder=1njMO20WaretNaZ7NxnKGyu20N9VmhB6K&pli=1#bookmark=id.8h2uikmachsj